ecl/hthor/hthor.cpp

```diff
@@ -8144,10 +8145,16 @@ void CHThorDiskReadBaseActivity::ready()
             if (publishedMeta)
                 publishedCrc = props.getPropInt("@formatCrc");
         }
+        dFile->getLogicalName(traceName);
```

Comment:
should traceName default to hthor-diskread if !dFile?

Reply:
Can do